include/z64animation.h

```diff
@@ -324,4 +322,89 @@ typedef s32 (*OverrideKeyframeDrawScaled)(struct PlayState* play, SkeletonInfo*
+void SkelAnime_InitPlayer(struct PlayState* play, SkelAnime* skelAnime, FlexSkeletonHeader* skeletonHeaderSeg, PlayerAnimationHeader* animation, s32 flags, void* jointTableBuffer, void* morphTableBuffer, s32 limbBufCount);
+void PlayerAnimation_SetUpdateFunction(SkelAnime* skelAnime);
+s32 PlayerAnimation_Update(struct PlayState* play, SkelAnime* skelAnime);
+void Animation_SetMorph(struct PlayState* play, SkelAnime* skelAnime, f32 morphFrames);
+void PlayerAnimation_Change(struct PlayState* play, SkelAnime* skelAnime, PlayerAnimationHeader* animation, f32 playSpeed, f32 startFrame, f32 endFrame, u8 mode, f32 morphFrames);
+void PlayerAnimation_PlayOnce(struct PlayState* play, SkelAnime* skelAnime, PlayerAnimationHeader* animation);
+void PlayerAnimation_PlayOnceSetSpeed(struct PlayState* play, SkelAnime* skelAnime, PlayerAnimationHeader* animation, f32 playSpeed);
+void PlayerAnimation_PlayLoop(struct PlayState* play, SkelAnime* skelAnime, PlayerAnimationHeader* animation);
+void PlayerAnimation_PlayLoopSetSpeed(struct PlayState* play, SkelAnime* skelAnime, PlayerAnimationHeader* animation, f32 playSpeed);
+void PlayerAnimation_CopyJointToMorph(struct PlayState* play, SkelAnime* skelAnime);
+void PlayerAnimation_CopyMorphToJoint(struct PlayState* play, SkelAnime* skelAnime);
+void PlayerAnimation_LoadToMorph(struct PlayState* play, SkelAnime* skelAnime, PlayerAnimationHeader* animation, f32 frame);
+void PlayerAnimation_LoadToJoint(struct PlayState* play, SkelAnime* skelAnime, PlayerAnimationHeader* animation, f32 frame);
+void PlayerAnimation_InterpJointMorph(struct PlayState* play, SkelAnime* skelAnime, f32 weight);
+void PlayerAnimation_BlendToJoint(struct PlayState* play, SkelAnime* skelAnime, PlayerAnimationHeader* animation1, f32 frame1, PlayerAnimationHeader* animation2, f32 frame2, f32 blendWeight, void* blendTableBuffer);
+void PlayerAnimation_BlendToMorph(struct PlayState* play, SkelAnime* skelAnime, PlayerAnimationHeader* animation1, f32 frame1, PlayerAnimationHeader* animation2, f32 frame2, f32 blendWeight, void* blendTableBuffer);
+void PlayerAnimation_EndLoop(SkelAnime* skelAnime);
+s32 Animation_OnFrameImpl(SkelAnime* skelAnime, f32 frame, f32 updateRate);
+s32 PlayerAnimation_OnFrame(SkelAnime* skelAnime, f32 frame);
+void SkelAnime_Init(struct PlayState* play, SkelAnime* skelAnime, SkeletonHeader* skeletonHeaderSeg, AnimationHeader* animation, Vec3s* jointTable, Vec3s* morphTable, s32 limbCount);
+void SkelAnime_InitFlex(struct PlayState* play, SkelAnime* skelAnime, FlexSkeletonHeader* skeletonHeaderSeg, AnimationHeader* animation, Vec3s* jointTable, Vec3s* morphTable, s32 limbCount);
+void SkelAnime_InitSkin(struct GameState* gameState, SkelAnime* skelAnime, SkeletonHeader* skeletonHeaderSeg, AnimationHeader* animation);
+void SkelAnime_SetUpdate(SkelAnime* skelAnime);
+s32 SkelAnime_Update(SkelAnime* skelAnime);
+void SkelAnime_AnimateFrame(SkelAnime* skelAnime);
```

Comment:
I don't think all of these functions need to be in this header, some can in internal to their files. `SkelAnime_AnimateFrame` for example looks to be internal to `z_skelanime.c`. Would you mind doing a parse through these?